media_service/mediastore.py

```diff
@@ -51,12 +78,14 @@ class MediaStoreUpload:
             media_store_instance = media_store_upload.save()
     '''
     VALID_IMAGE_EXTENSIONS = ('jpg', 'gif', 'png')
+    VALID_ZIP_EXTENSIONS = ('zip',)
```

Comment:
no. It was, and then yesterday when I was refactoring I thought I might end up using it again, but I didn't. So no, it should be removed.